fighters/dedede/src/status/special_lw.rs

```diff
@@ -9,6 +9,88 @@ unsafe extern "C" fn special_lw_jump_squat_exec(fighter: &mut L2CFighterCommon)
     return 0.into();
 }
 
+unsafe extern "C" fn special_lw_wait_pre(fighter: &mut L2CFighterCommon) -> L2CValue{
+    StatusModule::change_status_force(fighter.module_accessor, *FIGHTER_DEDEDE_STATUS_KIND_SPECIAL_LW_ATTACK, true);
```

Comment:
instead of using change_status_force, please do the following:
```rs
unsafe extern "C" fn special_lw_wait_pre(fighter: &mut L2CFighterCommon) -> L2CValue{
    StatusModule::set_status_kind_interrupt(fighter.module_accessor, *FIGHTER_DEDEDE_STATUS_KIND_SPECIAL_LW_ATTACK);
    1.into()
}
```

This will skip running the pre status entirely and go straight to the ATTACK status.